core/store/tx_manager_test.go

```diff
@@ -281,39 +273,31 @@ func TestTxManager_CreateTx_NonceTooLowReloadSuccess(t *testing.T) {
 		{"parity", "Transaction with the same hash was already imported"},
 	}
 
-	for _, test := range tests {
+	for _, tt := range tests {
+		test := tt
```

Comment:
I dont think this reassignment is needed

Reply:
I believe it is: https://golang.org/pkg/testing/ go captures the final state of the variable in a for loop in the closure without it.

Reply:
good catch